python/tvm/exec/popen_worker.py

```diff
@@ -49,6 +50,8 @@ def main():
         reader = os.fdopen(int(sys.argv[1]), "rb")
         writer = os.fdopen(int(sys.argv[2]), "wb")
 
+    logging.basicConfig(level=logging.INFO)
```

Comment:
logging is not used in this file. Is this expected?

Reply:
yes, we want to enable logging when functions being pushed to the worker contains logging logic.